internal/server/run.go

```diff
@@ -49,5 +55,5 @@ func run(c config.Config) (err error) {
 		return nil
 	}
 
-	return server.ListenAndServe(c.Listen, c)
+	return server.ListenAndServe(c.Listen, c, fsCache)
```

Comment:
So rather than passing `c.Listen` and `c`, we could just pass `c` and get the address from it.

Reply:
Good catch! I fixed it.
https://github.com/aquasecurity/trivy/pull/398/commits/4ee45ce93c5b6fdeeee6c2bff7e961d49eaadd28